cli/server/server.go

```diff
@@ -240,6 +241,16 @@ func restoreDB(ctx *cli.Context) error {
 	defer prometheus.ShutDown()
 	defer pprof.ShutDown()
 
+	var start uint32
+	if regexp.MustCompile(`chain.\d+.acc`).MatchString(ctx.String("in")) {
```

Comment:
NACK. Depending on file name is the worst thing we could do, please add a flag for incremental files.